### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by luckylooke**
fixing typo "public" instead of "pubilc"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ import { generateImage } from "jsdom-screenshot";
 
 // add some content to jsdom (this could also be React or any other library!)
 const div = document.createElement("div");
-div.innerText = "Hello World";
+div.innerHTML = "Hello World";
```
Comment (dferber90) [inline README.md:57]: `innerText` should stay here https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/innerText

**Review by dferber90 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ options = {
   // Options used to take a screenshot (puppeteer.screenshot(options))
   screenshot: {},
   // An array of folders containing static files to be served
-  serve: ["pubilc", "assets"],
+  serve: ["public", "assets"],
```
Comment (dferber90) [inline README.md:119]: Thanks, good catch!

File: README.md (hunk 1 above)
Comment (luckylooke) [reply README.md:57]: I don't know how you tested but it fails on my local due to innerText is not implemeted in jsdom, see discussion here: https://github.com/jsdom/jsdom/issues/1245

Comment (dferber90) [reply README.md:57]: Ah I don't remember how I tested it. With that context the change makes sense. Thanks!

**Review by dferber90 — APPROVED** [review]

**Pull request merged**